### ISSUE TITLE
Bugs, mis-documentations

### ISSUE THREAD

**Issue by jazzgil** [mrgadget/node-red-contrib-eztimer#21]
Spent a few hours trying to get this node to work with programmatic input. Ran into too many problems. From diving into the code, came up with the following:

1. Doc error: "onpayload" is actually "onvalue". Also found alias "triggervalue".
2. Doc error: "offpayload" is actually "offvalue".
3. ontopic / offtopic are completely unused. No actual way to set output topic.
4. No way to set tag programmatically.

I appreciate your efforts.

**Comment by mrgadget** [comment]
Hey @jazzgil, thanks for taking the time to submit an issue.  All clear, concise points.

1. (and 2) You had me stumped on the documentation ones but then I realised you're right - I've only been updating the readme.md for the package, not the HTML for the node help itself, so thanks for that too.
3. You're right! (again) - I've now re-introduced the topic value to the output message - existing programmatic code to set this will work.
4. Tag was a means to an end for a way I was using eztimer myself and I had no requirement to update programmatically, so that's why it is how it is.  That being said, super easy tweak to add it to the list of params so I've done that too.  And added it to _both_ docs.

I've just pushed 1.1.6 (with all these fixes) to NPM - give it a few hours and it should show as an available upgrade in node-red.  Let me know how you get on.
